docs/recipes.adoc

```diff
@@ -5,6 +5,8 @@
 Welcome to the recipes section of the Couchbase Shell `cbsh` documentation.
 Here you can find some examples of the more complicated tasks that can be performed using `cbsh`.
```

Comment:
I know this wasn't changed here, but maybe…

"Here you can find some examples of how powerful tasks can be performed in a set of pipelined statements using `cbsh`." or something like that?